### PR TITLE
Make AIRHANDLER autosized heat pump capacities consistent

### PR DESCRIPTION
**Pull request by nealkruis**
## Description

Addresses #239.

## Author Progress Checklist:

- [x] Open draft pull request
    - [x] Make title clearly understandable in a standalone change log context
    - [x] Assign yourself the issue
    - [x] Add at least one label (enhancement, bug, or maintenance)
    - [x] Link the issue(s) addressed by this PR (under "Development" in the sidebar menu) 
- [x] Make code changes (if you haven't already)
- [x] Self-review of code
    - [ ] My code follows the style guidelines of this project
    - [ ] I have added comments to my code, particularly in hard-to-understand areas
    - [ ] I have only committed the necessary changes for this fix or feature
    - [ ] I have made corresponding changes to the documentation
    - [ ] My changes generate no new warnings
    - [ ] I have ensured that my fix is effective or that my feature works as intended by:
        - [ ] exercising the code changes in the test framework, and/or
        - [ ] manually verifying the changes (as explained in the the pull request description above)
    - [ ] My changes pass all local tests
    - [ ] My changes successfully passes CI checks
    - [ ] Add any unit test for proof and documentation.
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
- [x] Move pull request out of draft mode and assign reviewers
- [ ] Iterate with reviewers until all changes are approved
    - [ ] Make changes in response to reviewer comments
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
    - [ ] Re-request review in GitHub

## Reviewer Checklist:

 - [ ] Read the pull request description
 - [ ] Perform a code review on GitHub
 - [ ] Confirm all CI checks pass and there are no build warnings
 - [ ] Pull, build, and run automated tests locally
 - [ ] Perform manual tests of the fix or feature locally
 - [ ] Add any review comments, if applicable
 - [ ] Submit review in GitHub as either
     - [ ] Request changes, or
     - [ ] Approve
 - [ ] Iterate with author until all changes are approved

### CODE REVIEW

File: src/CNLOADS.CPP [hunk 1]
```diff
@@ -4499,9 +4469,9 @@ RC RSYS::rs_SetupASHP()		// set ASHP defaults and derived parameters
 	// inter-default cap95 / cap47
 	//   at least one is present (see rs_CkF)
 	if (!IsSet( RSYS_CAP95))
-		rs_cap95 = rs_Cap95FromCap47ASHP( rs_cap47);
+		ashpCap95FromCap47( rs_cap47, IsSet(RSYS_CAPRAT9547), rs_capRat9547);
```
Comment (chipbarnaby) [inline src/CNLOADS.CPP:4472]: Should this not be rs_cap95 = ashpCap95FromCap47(  ...) ?

File: src/CNLOADS.CPP [hunk 2]
```diff
@@ -4499,9 +4469,9 @@ RC RSYS::rs_SetupASHP()		// set ASHP defaults and derived parameters
...
 	else if (!IsSet( RSYS_CAP47))
-		rs_cap47 = rs_Cap47FromCap95ASHP( rs_cap95);
+		ashpCap47FromCap95( rs_cap95, IsSet(RSYS_CAPRAT9547), rs_capRat9547);
```
Comment (chipbarnaby) [inline src/CNLOADS.CPP:4474]: Ditto rs_cap47 = ashpCap47FromCap95( ... )

Alarming that tests did not pick this up.

File: src/HVAC.CPP [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+#include "cnglob.h"
+
+//-----------------------------------------------------------------------------
+float CoolingSHR(		// derive cooling sensible heat ratio
+	float tdbOut,		// outdoor dry bulb, F
+	float tdbCoilIn,	// coil entering dry bulb, F
+	float twbCoilIn,	// coil entering wet bulb, F
+	float vfPerTon)		// coil air flow std air cfm/ton
+
+// return: SHR under current conditions
```
Comment (chipbarnaby) [inline src/HVAC.CPP:17]: Document where this comes from.

**Review by chipbarnaby — CHANGES_REQUESTED** [review]
See comments.

Perhaps we need to add a test that covers the cross estimation of capacity?

File: src/CNLOADS.CPP (hunk 1 above)
Comment (nealkruis) [reply src/CNLOADS.CPP:4472]: Good catch. Don't know how I missed that. I think the scenario here is pretty unlikely (and part of why I didn't add something analogous to AIRHANDER, where currently both capacities are required).

**Pull request merged**